### PR TITLE
Add kubernetes pip dependecy handling to renovate

### PR DESCRIPTION
**Pull request by Rozzii**
This dependency is used by one of Ansible's collections/plugins
named kubernetes.core.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -61,6 +64,12 @@
        "depNameTemplate": "ansible",
        "datasourceTemplate": "pypi"
      },
+     {
+       "fileMatch": ["^vm-setup/roles/packages_installation/defaults/main.yml$"],
+       "matchStrings": ["kbuernetes==(?<currentValue>.*?)"],
```
Comment (fmuyassarov) [inline renovate.json:69]:
```suggestion
       "matchStrings": ["kubuernetes==(?<currentValue>.*?)"],
```

**Review by fmuyassarov — CHANGES_REQUESTED** [review]

File: renovate.json (hunk 1 above)
Comment (Rozzii) [reply renovate.json]: Thanks, fixed it.

**Review by Sunnatillo — COMMENTED** [review]
Looks good to me

**Review by fmuyassarov — APPROVED** [review]

**Review by kashifest — APPROVED** [review]

**Pull request merged**